src/main/resources/assets/clientcommands/lang/en_us.json

```diff
@@ -43,7 +43,7 @@
   "commands.ccheatcrackrng.success": "Player RNG cracked: %d -> %d",
   "commands.ccheatcrackrng.java14": "This command no longer works on Java 14+!",
 
-  "commands.cenchant.expectedWithWithout": "Expected \"with\"/\"without\"",
+  "commands.cenchant.expectedWithWithoutExactly": "Expected \"with\"/\"without\"/\"exactly\"",
```

Comment:
Removing translations requires them to be removed in all the different language files. When I change a translation like this, I usually just delete the translated ones and let the translators re-add it later.